README.md

```diff
@@ -34,19 +42,29 @@ collections:
 - name: community.general
 ```
 
+Note that if you install the collection manually, it will not be upgraded automatically when you upgrade the Ansible package. To upgrade the collection to the latest available version, run the following command:
+
+```bash
+ansible-galaxy collection install community.general --upgrade
+```
+
+You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax where `X.Y.Z` can be any [available version](https://galaxy.ansible.com/community/general):
+
+```bash
+ansible-galaxy collection install community.general:==X.Y.Z
+```
+
 See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
 
 ## Contributing to this collection
 
-If you want to develop new content for this collection or improve what is already here, the easiest way to work on the collection is to clone it into one of the configured [`COLLECTIONS_PATH`](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths), and work on it there.
+The content of this collection is made by good people like you, a community of individuals collaborating on making the world better through developing automation software.
 
-For example, if you are working in the `~/dev` directory:
+All types of contributions are very welcome.
 
-```
-cd ~/dev
-git clone git@github.com:ansible-collections/community.general.git collections/ansible_collections/community/general
-export COLLECTIONS_PATH=$(pwd)/collections:$COLLECTIONS_PATH
-```
+You don't know how to start? Refer to our [contribution guide](https://github.com/ansible-collections/community.general/blob/main/CONTRIBUTING.md)!
+
+The current maintainers are listed in the [commit-rights.md](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md#people) file. Don't hesitate to reach them out mentioning in the proposals.
```

Comment:
"Don't hesitate to reach out to them by mentioning them in the proposals."

Reply:
@cybette how about the new wording? PTAL